src/main/java/de/srendi/advancedperipherals/common/items/MemoryCardItem.java

```diff
@@ -28,21 +29,27 @@ public boolean isEnabled() {
     @Override
     public void appendHoverText(ItemStack stack, @Nullable Level levelIn, List<Component> tooltip, TooltipFlag flagIn) {
         super.appendHoverText(stack, levelIn, tooltip, flagIn);
-        if (stack.getOrCreateTag().contains("owner"))
-            tooltip.add(EnumColor.buildTextComponent(Component.translatable("item.advancedperipherals.tooltip.memory_card.bound", stack.getOrCreateTag().getString("owner"))));
-
+        CompoundTag data = stack.getOrCreateTag();
+        // TODO: remove the owner name field
```

Comment:
Why is this marked as TODO? For backwards compatibility?
If so, please mark a version after the TODO or improve the todo comment to explain that
Like `TODO - 1.21: xyz`